README.md

```diff
@@ -52,34 +46,30 @@ The authorization flow and parsing of the xml data must be handled by each platf
 
 # Technical Design
 
-## shared
-
-## Android
-
 [Wi-Fi infrastructure](https://developer.android.com/guide/topics/connectivity/wifi-infrastructure)
 "On Android 10 and higher, the Wi-Fi infrastructure includes the Wi-Fi suggestion API for internet
 connectivity [...]. On Android 11 and higher, the Settings Intent API enables you to ask the user to
 approve adding a saved network or Passpoint configuration."
 
+In general the app will use the following APIs for createing the WiFI configuration:
+
```

Comment:
Am I missing the API list?

Reply:
It is there below, I am just reusing the existing documentation that's why it doesn't show up as new